mongo/mongo.go

```diff
@@ -238,13 +238,20 @@ var (
 		Patch:         "",
 		StorageEngine: WiredTiger,
 	}
-	// Mongo406wt represents 'mongodb-server-core' at version 4.0.6 with WiredTiger
+	// Mongo406wt represents snap 'juju-db' at version 4.0.x with WiredTiger
+	// XXX: (jam) This is the only Mongo declaration that declares a Point
+	//  release. It doesn't seem the right place to do so. We currently have to,
+	//  because the K8s pod spec requires it but
+	//  a) It doesn't match the current snap version
+	//  b) It prevents our normal "update the package to get an updated mongo without
+	//     rebuilding Juju".
+	//  c) It will get *real* messy if we support a range of 4.0.* releases.
```

Comment:
I've updated my PR to omit the point version.
It will now just pull juju-db:4.0 (or whatever)
We can tag whatever upstream point version we want as 4.X